.github/workflows/sync_meta.yml

```diff
@@ -6,8 +6,8 @@ on:
   schedule:
   - cron: "0 0 * * *"
   push:
-    paths:
-    - '.github/**'
+    branches:
```

Comment:
Do we need both the path _and_ branch conditions, or just this one?

Reply:
I'm not sure if GitHub uses branch **AND** path or branch **OR** path. So I went with just branch. The sync workflow won't open PRs if the files have not changed so it should be fine.